projects/element-ng/tabs-next/si-tab-next.component.ts

```diff
@@ -0,0 +1,70 @@
+/**
+ * Copyright Siemens 2016 - 2025.
+ * SPDX-License-Identifier: MIT
+ */
+import { NgClass } from '@angular/common';
+import { ChangeDetectionStrategy, Component, model, OnDestroy } from '@angular/core';
+import { SiIconNextComponent } from '@siemens/element-ng/icon';
+import { SiTranslateModule } from '@siemens/element-translate-ng/translate';
+
+import { SiTabNextBaseDirective } from './si-tab-next-base.directive';
+
+/** @experimental */
+@Component({
+  selector: 'si-tab-next',
+  templateUrl: './si-tab-next.component.html',
+  styleUrl: './si-tab-next.component.scss',
+  changeDetection: ChangeDetectionStrategy.OnPush,
+  imports: [NgClass, SiIconNextComponent, SiTranslateModule],
+  host: {
+    '[class.active]': 'active()',
+    '[attr.aria-selected]': 'active()',
+    '(click)': 'selectTab()',
+    '(keydown.enter)': 'selectTab()'
+  }
+})
+export class SiTabNextComponent extends SiTabNextBaseDirective implements OnDestroy {
+  /**
+   * Whether the tab is active or not.
+   * If set to `true`, the tab will be selected and its content will be displayed.
+   * @defaultValue false
+   * */
+  override readonly active = model.required<boolean>();
```

Comment:
sorry, @chintankavathia I was failing with my last suggestion. No need for required here:
```suggestion
  override readonly active = model(false);
```

if this blocks the usage undefined, than this:
```ts
  override readonly active = model<boolean>(false);
```